dockerfiles/init/manifests/che.env

```diff
@@ -56,6 +56,13 @@
 #CHE_DEBUG_SUSPEND=false
 #CHE_DEBUG_PORT=8000
 
+# Che master logs
+#
+# Choose variant of Che master logs encoding and storing. Default is `plaintext`.
+# Another supported value is `json` which makes Che master produce JSON encoded logs
+# in system output of PID 1 process of container. Useful for storing logs in systems such as Logstash.
+# CHE_LOGS_APPENDERS_IMPL=plaintext
```

Comment:
Shouldn't default value be added into che.properties?

Reply:
No, it is not read from properties